pom.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>103</version>
+    </parent>
+
+    <groupId>io.prestosql.yugabyte</groupId>
+    <artifactId>yugabyte-driver</artifactId>
+    <version>3.6.0-2-SNAPSHOT</version>
+
+    <description>Shaded version of Yugabyte Java Driver for Apache Cassandra</description>
+    <url>https://github.com/prestosql/presto-yugabyte-driver</url>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <!-- see original projects for developer information -->
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/prestosql/presto-yugabyte-driver.git</connection>
+        <url>https://github.com/prestosql/presto-yugabyte-driver</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <properties>
+        <project.build.targetJdk>11</project.build.targetJdk>
+        <shadeBase>io.prestosql.yugabyte.\$internal</shadeBase>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.yugabyte</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <version>3.2.0-yb-19</version>
+            <scope>runtime</scope>
+        </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>19.0</version>
```

Comment:
Can you please add a comment why this particular Guava version?
(it's pretty old, so explanation is needed)

perhaps
```
<!-- Use Guava version dependent upon by cassandra-driver-core. Otherwise version provided by airbase would be used. -->
```